run_dir/design/flowcells.html

```diff
@@ -9,6 +9,9 @@
 
 {% block stuff %}
 
+<script type="text/javascript" src="/static/js/base.js"></script>
+<script src="/static/js/marked.min.js"></script>
```

Comment:
Both of these js-files are imported in the base.html template. But maybe they are not available right away since they are at the bottom. Can it the markdown parsing be triggered after the page has loaded?

Reply:
Yes - this is typically done using the following jQuery syntax:

```javascript
$(function(){
   // write code here that will be executed when the entire page has been loaded.
});
```